docs/app/Examples/views/Card/Variations/ColumnCount.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Card } from 'stardust'
+
+const src = 'http://semantic-ui.com/images/wireframe/image.png'
+
+const ColumnCountCards = () => (
+  <Card.Group itemsPerRow={6}>
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
+    <Card raised href='#column-count' image={src} />
```

Comment:
Let's remove the `href` here.  Ideally, we show as little features as possible for the example at hand.  In this case, we are showing the `itemsPerRow` prop.  The `raised` and `image` are OK.